lib/lexer/src/lexer.rs

```diff
@@ -448,7 +454,7 @@ impl Lexer {
                 token.push(self.eat_char());
 
                 // Consume ('_'? digitbinary)*.
-                self.consume_digits(self.digit_binary.clone(), &mut token);
+               self.consume_digits(self.digit_binary.clone(), &mut token);
```

Comment:
Minor nitpick. Indentation is wrong here.

Reply:
oops